src/mscorlib/src/System/Type.cs

```diff
@@ -1182,7 +1182,7 @@ public bool IsArray
             get { return IsArrayImpl(); }
         }
 
-        internal virtual bool IsSzArray
+        public virtual bool IsSZArray
         {
             [Pure]
             get { return false; }
```

Comment:
@JonHanna  The default implemention now needs to be "throw new NotImplementedException();"